omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/impl/BaseDelegatingResource.java

```diff
@@ -118,14 +119,18 @@ public Model getGETModel(Representation rep) {
 	protected volatile List<DelegatingSubclassHandler<T, ? extends T>> subclassHandlers;
 	
 	/**
-	 * Default constructor will set propertiesIgnoredWhenUpdating to include "display", "links", and
+	 * Default constructor will set propertiesIgnoredOnPostAndPut to include "display", "links", and
 	 * "resourceVersion"
 	 */
 	protected BaseDelegatingResource() {
-		propertiesIgnoredWhenUpdating.add("display");
-		propertiesIgnoredWhenUpdating.add("links");
-		propertiesIgnoredWhenUpdating.add("auditInfo");
-		propertiesIgnoredWhenUpdating.add(RestConstants.PROPERTY_FOR_RESOURCE_VERSION);
+		propertiesIgnoredOnPostAndPut.add("display");
+		propertiesIgnoredOnPostAndPut.add("links");
+		propertiesIgnoredOnPostAndPut.add("auditInfo");
+		propertiesIgnoredOnPostAndPut.add(RestConstants.PROPERTY_FOR_RESOURCE_VERSION);
+		
+		for (int i = 7; i < 16; i++) {
```

Comment:
This is also unnecessary, it's the test data that is wrong, the higher addresses fields 7-15 were added in 2.0